var/spack/repos/builtin/packages/intel-parallel-studio/package.py

```diff
@@ -20,6 +20,7 @@ class IntelParallelStudio(IntelPackage):
     # in the 'intel' package.
 
     # Cluster Edition (top tier; all components included)
+    version('cluster.2019.5',      sha256='c03421de616bd4e640ed25ce4103ec9c5c85768a940a5cb5bd1e97b45be33904', url='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/15809/parallel_studio_xe_2019_update5_cluster_edition.tgz')
```

Comment:
Could you also update professional edition?

Reply:
I searched the download URL in Intel registration center and only found cluster edition, cluster edition and individual mkl, mpi packages. I didn't see a link for professional edition.